### PR TITLE
Add NumberOfRatings to WorkshopCard

### PR DESCRIPTION
**Pull request by alrom**

### CODE REVIEW

**Review by andriitsylia — APPROVED** [review]

File: OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs [hunk 1]
```diff
@@ -195,10 +195,13 @@ public async Task<IEnumerable<WorkshopCard>> GetPopularWorkshopsFromDatabase(int
     private async Task<List<WorkshopCard>> GetWorkshopsWithAverageRating(List<WorkshopCard> workshopsCards)
     {
         var averageRatings = await averageRatingService.GetByEntityIdsAsync(workshopsCards.Select(p => p.WorkshopId)).ConfigureAwait(false);
+        AverageRatingDto averageRatingDto = null;
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs]: (Code style) This row looks bit excessive. You could just say `var averageRatingDto = averageRatings?....` in `foreach` cycle body and that's it. Don't worry, no performance penalties will be provided.

File: OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs [hunk 2]
```diff
@@ -768,9 +768,13 @@ private async Task<List<T>> GetWorkshopsWithAverageRating<T>(List<T> workshops)
     {
         var averageRatings = await averageRatingService.GetByEntityIdsAsync(workshops.Select(p => p.WorkshopId)).ConfigureAwait(false);
 
+        AverageRatingDto averageRatingDto = null;
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs]: (Same code style comment) Variable definition should be as close as possible to variable usage. Please get rid of this line and use `var averageRatingDto = ...` in cycle body

File: OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs (hunk 1 above)
Comment (alrom) [reply OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs]: Ok. Done.

File: OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs (hunk 2 above)
Comment (alrom) [reply OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs]: Done

**Review by VadymLevkovskyi — APPROVED** [review]

**Pull request merged**